File: executable/r/stacked-histogram.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

x0 = rnorm(500)
x1 = rnorm(500)+1

trace1 <- list(
  x = x0, 
  type = "histogram"
)
trace2 <- list(
  x = x1, 
  type = "histogram"
)

layout <- list(barmode = "stacked")



response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="stacked-histogram", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/julia/stacked-histogram.jl
using Plotly

using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

x0 = randn(500)
x1 = randn(500)+1

trace1 = [
  "x" => x0, 
  "type" => "histogram"
]
trace2 = [
  "x" => x1, 
  "type" => "histogram"
]
data = [trace1, trace2]
layout = ["barmode" => "stacked"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "stacked-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]